### PR TITLE
Adding default values for record_latency and language

### PR DESCRIPTION
**Pull request by tmatsuo**
@nkubala @liyanhui1228

### CODE REVIEW

**Comment by tmatsuo** [comment]
Otherwise, the integration test driver fails with:

```
TypeError: deploy_app() takes exactly 6 arguments (4 given)
```

File: integration_tests/testsuite/deploy_app.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def _record_latency_to_bigquery(deploy_latency, language):
 
 
 def deploy_app(base_image, builder_image, appdir,
-               yaml, record_latency, language):
+               yaml, record_latency = False, language = None):
```
Comment (nkubala) [inline integration_tests/testsuite/deploy_app.py]: nit: could you remove the whitespace around the equals signs?

`record_latency=False, language=None`

**Comment by tmatsuo** [comment]
Done

**Comment by liyanhui1228** [comment]
LGTM

**Pull request merged**